src/ceph.in

```diff
@@ -600,12 +599,6 @@ def new_style_command(parsed_args, cmdargs, target, sigdict, inbuf, verbose):
             # Non interactive mode
             ret, outbuf, outs = do_command(parsed_args, target, cmdargs, sigdict, inbuf, verbose)
         else:
-            # Interactive mode (ceph cli)
```

Comment:
how do we get to remove this?  Is readline support now builtin?..

Reply:
Is just importing readline meant to have an effect?  It seems like e.g. pressing the up arrow to see last command already doesn't work, so I'm not sure what it was meant to be doing

Reply:
It had been the case that, when raw_input() was used, "import readline" made it support line-editing-and-recall features.  Perhaps the port to Py3 lost that behavior (as raw_input disappeared too)?

https://docs.python.org/2/library/functions.html#raw_input

Reply:
https://gist.github.com/dmick/90b43cb15464417870147d953d7a2412 works as I'd expect for me with py2 and py3.  ?

Reply:
Huh, yeah, that snippet does show importing readline having the side effect on raw_input, which is presumably why it was in ceph.in.  I'll reinstate it.  Thanks for setting me straight!

Reply:
(my "pre-import" comment was about the IDE, which of course makes sense since it also supports commandline editing)